packages/bpk-component-graphic-promo/src/BpkGraphicPromo.js

```diff
@@ -0,0 +1,202 @@
+  addText(strapline);
+  addText(ctaText);
+
+  return text.join('. ');
+};
+
+const BpkGraphicPromo = (props: Props) => {
+  const {
+    className,
+    ctaText,
+    headline,
+    image,
+    invertVertically,
+    kicker,
+    onClick,
+    sponsor,
+    strapline,
+    textAlign,
+    textColor,
+  } = props;
+
+  // FIXME: Use useCallback() here when React is updated.
+  const onClickWrapper = (event: React.MouseEvent<HTMLElement>) => {
+    event.stopPropagation();
+    onClick();
+  };
+
+  const cardClasses = [getClassName('bpk-graphic-promo')];
+  if (className) {
+    cardClasses.push(className);
+  }
+
+  const containerClasses = [
+    getClassName('bpk-graphic-promo__container'),
+    getClassName(`bpk-graphic-promo__container--${textAlign}`),
+  ];
+  if (invertVertically) {
+    containerClasses.push(
+      getClassName('bpk-graphic-promo__container--inverted'),
+    );
+  }
+
+  const getTextClasses = (baseClass: string) => {
+    const classes = [getClassName(baseClass)];
+    classes.push(getClassName(`${baseClass}--${textAlign}`));
+
+    return classes.join(' ');
+  };
+
+  return (
+    <BpkCard
+      className={cardClasses.join(' ')}
+      style={{ backgroundImage: `url(${image})`, color: textColor }}
+      onClick={onClickWrapper}
+      aria-label={constructAriaLabel(props)}
+      padded={false}
+    >
+      <div className={containerClasses.join(' ')} aria-hidden>
+        {sponsor && (
+          <div className={getTextClasses('bpk-graphic-promo__sponsor-content')}>
+            <BpkText
+              tagName="span"
+              className={getClassName('bpk-graphic-promo__sponsor-label')}
+            >
+              {sponsor.label}
+            </BpkText>
+            <img
+              className={getClassName('bpk-graphic-promo__sponsor-logo')}
+              alt={sponsor.altText}
+              src={sponsor.logo}
+            />
+          </div>
+        )}
+        <div className={getTextClasses('bpk-graphic-promo__promo-content')}>
+          {!sponsor && kicker && (
+            <BpkText
+              tagName="span"
+              className={getClassName('bpk-graphic-promo__kicker')}
+            >
+              {kicker}
+            </BpkText>
+          )}
+          <BpkText
+            tagName="h2"
+            className={getClassName('bpk-graphic-promo__headline')}
+          >
+            {headline}
+          </BpkText>
+          {strapline && (
+            <BpkText
+              tagName="p"
+              className={getClassName('bpk-graphic-promo__strapline')}
+            >
+              {strapline}
+            </BpkText>
+          )}
+          <BpkButton
+            className={getClassName('bpk-graphic-promo__cta')}
+            onClick={onClickWrapper}
+          >
+            {ctaText}
+          </BpkButton>
+        </div>
+      </div>
+    </BpkCard>
+  );
+};
+
+BpkGraphicPromo.propTypes = {
+  className: PropTypes.string,
+  kicker: PropTypes.string,
```

Comment:
Can we review this prop name and maybe come up with a different name? As looking at it from the code and docs I would have no idea what `kicker` means or refers to. Prop names should be self explanatory as to what they do from a glance

Reply:
The names are coming straight from the design document. It is an optional piece of text we put above the headline. Happy to use a different name, if you have something else in mind. 🙂

Reply:
Tagline?

Reply:
Updated. 🙂